giskard/visualization/templates/scan_report/markdown/huggingface.md

```diff
@@ -0,0 +1,38 @@
+Hey Team!🤗✨ 
+
+We’re thrilled to share some amazing evaluation results that’ll make your day!🎉📊
+
+{% for view in groups -%}
+<details>
+<summary>👉{{ view.group.name }} issues ({{ view.issues|length }})</summary>
+
+{% for issue in view.issues -%}
+
+| Vulnerability | Level | Data slice | Metric | Transformation | Deviation | Description |
+|---------------|-------|------------|--------|----------------|-----------|-------------|
+| {{ view.group.name }} | {{ issue.level.value }} | {{ issue.slicing_fn if issue.slicing_fn else "—" }} | {% if "metric" in issue.meta %}{{ issue.meta.metric }} = {{ issue.meta.metric_value|format_metric }}{% else %} "—" {% endif %} | {{ issue.transformation_fn if issue.transformation_fn else "—" }} | {{ issue.meta["deviation"] if "deviation" in issue.meta else "—" }} | {{ issue.description }} |
```

Comment:
How about extracting description from the table and put it before the example?
<img width="1244" alt="image" src="https://github.com/Giskard-AI/giskard/assets/8311300/4985b863-3e14-4c57-adb4-fbedb71e2df6">
We can thus gain some spaces for each cells

Reply:
How about making HF template more vivant? Like:
Major 🔴
Minor 🟡

Reply:
✅ Updated the colors and added emojis for issue levels.